src/main/java/xbot/common/controls/sensors/XGamepad.java

```diff
@@ -0,0 +1,70 @@
+package xbot.common.controls.sensors;
+
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.math.XYPair;
+
+public abstract class XGamepad extends XJoystick {
+
+	// All of the "Left Joystick" inversions are kept track of by the base XJoystick class.
+	// We just need to redirect any calls to the base class.
+	private boolean xRightInverted = false;
+    private boolean yRightInverted = false;
```

Comment:
Indentation

Reply:
I think something is wrong with whitespace on one of my machines. I just did a Source->Correct Indentation, we'll see how that's interpreted by github.